README.md

```diff
@@ -203,25 +206,62 @@ invalid modifier is used.
 ### Applying Modifiers
 
 Applying modifiers when rendering the component is as simple as providing a
-`modifiers` prop. The prop should be an array of strings that correspond to keys
-in the modifier configuration object applied to the component.
+`modifiers` prop. The value of this prop can be one either a string, an array of
+strings. Each string value should correspond to keys in the modifier
+configuration object applied to the component.
 
 ```jsx
 function Form() {
   return (
     <div>
       {/* ...the rest of form goes here... */}
-      {/* Render a button, and give it a `modifiers` prop with the desired modifiers. */}
+      {/* Render a button, and give it a `modifiers` prop with the desired modifier. */}
       <Button modifiers="success" />
+
+      {/* This is also perfectly valid, and will result in "stacked" modifiers. */}
+      <Button modifiers={['success', 'large']} />
     </div>
   );
 }
 ```
 
-In the example above, our button will have the `styles` from the `success`
-modifier applied.
+You can also apply the modifiers to be responsive. For this, the value of the
+`modifiers` prop should be an object where each value is either a string or
+array of strings that match a modifier name. Which set of modifiers is chosen
+will be based on the value of a `size` prop that you must also provide to the
+component.
+
+```jsx
+<Button
+  modifiers={{
+    small: 'disabled', // <-- will be applied when `getTheSizeFromSomewhere()` returns 'small'
+    medium: ['success', 'large'],
+  }}
+  size={getTheSizeFromSomewhere()}
+/>
+```
+
+**Note:**
+
+If you apply modifiers using the responsive technique and the value of `size`
+doesn't match a key in the object, the value of `_` will be applied by default.
+This is useful for applying special modifiers only on a single size.
+
+```jsx
+<Button
+  modifiers={{
+    _: 'disabled', // <-- will be applied for all size values _except_ 'medium'.
```

Comment:
Question: the `_` provides a fallback if none of the keys match the value passed to `size`. But what about common modifiers we want to pass to _all_ of the breakpoints?  For example:

```jsx
<Button
  modifiers={{
    _: 'disabled',
    small: ['foo', 'bar', 'success', 'micro'],
    medium: ['foo', 'bar', 'success, 'align_left', 'fontSize_large'],
  }}
...
```

Should we also have something providing for those common-to-all modifiers?  Maybe something like this...

```jsx
<Button
  modifiers={{
    _: 'disabled',
    '*': ['foo', 'bar', 'success']
    small: 'micro',
    medium: ['align_left', 'fontSize_large'],
  }}
...
```

🤷‍♂  at the same time, this seems like it's getting pretty complicated to use.

Reply:
I'd like to keep the shared modifiers responsibility on the devs using the tool. It's very easy to implement when building components.

```jsx
const commonModifiers = ['foo', 'bar', 'success'];
...
<Button
  modifiers={{
    _: 'disabled',
    small: [...commonModifiers, 'micro'],
    medium: [...commonModifiers, 'align_left', 'fontSize_large'],
  }}
/>
```

I think default key makes sense because it's not functionality you can easily get in userland. Common modifiers is simple in userland though, and the API might be surprising for users.